woocommerce/woocommerce-ios
# [Manual Taxes M3] Stored Tax Rate bottom sheet

Part of: #10681 


## Description

With this PR we implement the UI part when we have a stored tax rate and tap to Edit that tax rate. The design shows a bottom sheet, and since `presentationDetents` is only available on iOS16+, we have to create our own `AdaptiveSheet` view modifier that implements the `adaptiveSheetPresentationController` properties. We also had to handle orientation changes as it crashes in landscape due to the system not finding the detents. 

The bottom sheet doesn't match the design height, but since on iOS 15 the size are fixed and depend on the detents, I preferred to reuse the standard height instead of implementing our own custom classes.

## Testing instructions


1. Follow the steps of https://github.com/woocommerce/woocommerce-ios/pull/10695 to store a selected tax rate.
2. Tap on + to create a new order. Wait until the stored tax rate is fetched and applied to the order.
3. Tap on Edit Tax Rate Setting. See that the stored tax rate info is shown on the bottom sheet plus the other design components. (See screenshot below)

## Screenshots


<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/4ac0e943-bc17-4c0a-b2cb-117d7cec546a" width="375">

---
